src/styles/blocks/stars.scss

```
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+
+  &__star {
+    margin-right: 4px;
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
```

You can write this shorter
```suggestion
    background: url(../images/star.svg) center no-repeat;
```